modules/templates/DIRECT/menus.py

```diff
@@ -227,13 +231,13 @@ def inv():
 
         return M()(
                     M("Warehouses", c="inv", f="warehouse")(
-                        M("Create", m="create"),
-                        M("Import", m="import", p="create"),
+                        M("Create", m="create", restrict=[ADMIN, ORG_ADMIN, SUPPLY_COORDINATOR]),
+                        M("Import", m="import", p="create", restrict=[ADMIN, ORG_ADMIN, SUPPLY_COORDINATOR]),
```

Comment:
The above said, there is no need for looking up SUPPLY_COORDINATOR from system_roles, instead, you can simply specify a string here, i.e. `restrict = [ADMIN, ORG_ADMIN, "SUPPLY_COORDINATOR"]`. This will then be looked up from `auth_group.uuid` automatically, and it will also be cached, so the lookup happens both _lazy_ and only _once_.

Theoretically, this works with ADMIN and ORG_ADMIN too.